client/src/components/DestinationPageMainContent/DestinationPageMainContent.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const DestinationImageMapContainer = styled.div`
+  display: flex;
+  direction: row;
```

Comment:
Maybe I'm wrong: but isn't it supposed to be declared as  `flex-direction`?

Reply:
You are right! It was working because it is always row by default, but yeah it was not correct!